spec/Schema.spec.js

```diff
@@ -494,7 +494,8 @@ describe('SchemaController', () => {
 
   it('creates non-custom classes which include relation field', done => {
     config.database.loadSchema()
-    .then(schema => schema.addClassIfNotExists('_Role', {}))
```

Comment:
We could add a test, though, that makes sure `addClassIfNotExists` returns the existing schema, instead of an error, if the class already exists.